savant/config/json_resolver.py

```diff
@@ -16,7 +16,10 @@ def json_resolver(  # pylint:disable=unused-argument
     string."""
     logger.debug('Parsing param JSON "%s"', json_string)
     try:
-        parsed_conf_node = json.loads(json_string)
+        if json_string is not None:
+            parsed_conf_node = json.loads(json_string)
+        else:
+            return None
     except TypeError:
         logger.warning('JSON loads fail, returning None for "%s".', json_string)
         return None
```

Comment:
Is this to avoid seeing the warning when json str is None?

I think the following is a bit more readable, but up to you.

```python

try:
    parsed_conf_node = json.loads(json_string)
except TypeError:
    if json_string:
        logger.warning('JSON loads fail, returning None for "%s".', json_string)
    return None
```